task1.py

```diff
@@ -0,0 +1,73 @@
+"""
+Include tasks for basic data types and  exception handling
+"""
+
+import sys
+
+
+def remove_all_except_first(word):
+    """
+    Remove all characters in a string except first occurrence of that character
+    """
+    no_duplicates = ''
+    for character in word:
+        # check for both upper and lower cases
+        if character not in no_duplicates and str(character).upper() not in no_duplicates.upper():
```

Comment:
for case insensitive lookup, we can make word lower right at the beginning instead of later in conditions. 
```word = word.lower()```

Reply:
If we do that it will not include upper case letters in input string.
For Ex: TechCity -> techcity
now the the output will be techiy instead of Techiy

Reply:
ah ok, even if you want to retain input string characters case then ```if character.lower() not in no_duplicates.lower()``` is enough.

Reply:
got it